Client/src/index.html

```diff
@@ -2,16 +2,10 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>
-      Starting React
-    </title>
+    <title>Starting React</title>
   </head>
 
   <body>
-
     <div id="root"></div>
-
```

Comment:
I am a fan of the html plugins for webpack. The Udemy course is using a brittle approach.